ctk/common/src/main/java/org/codice/compliance/utils/sign/SimpleSign.java

```diff
@@ -279,17 +276,6 @@ public void validateSignature(Signature signature) throws SignatureException {
     }
 
     validateSignatureAndSamlKey(signature, samlKeyInfo);
-
```

Comment:
❓ Why is this being taken out?

Reply:
It's being taken out because otherwise it requires users to put their IdP's cert into our keystore which is painful and unnecessary. It's unnecessary because this is a testing tool that should only be run in a test environment. Trusting the certs that were used for signing is significantly less important in such a scenario.